pallas-network/src/miniprotocols/localstate/queries_v16/mod.rs

```diff
@@ -148,3 +175,19 @@ pub async fn get_stake_distribution(
 
     Ok(result)
 }
+
+pub async fn get_utxo_by_address(
+    client: &mut Client,
+    era: u16,
+    address: String,
+) -> Result<UTxOByAddress, ClientError> {
+    let addr: Address = Address::from_bech32(&address).unwrap();
```

Comment:
we shouldn't handle bech32 conversion here. If the user wants to use bech32, it should happen higher in the stack, outside of the network crate.

Instead of a string, accept the address bytes directly.